packages/esm-patient-conditions-app/src/conditions/conditions-overview.scss

```diff
@@ -10,7 +10,7 @@
 .divider {
   width: 1px;
   height: layout.$spacing-05;
-  color: colors.$gray-20;
+  color: $ui-05;
```

Comment:
Ditto

Reply:
When imported as *, it should function properly without requiring the `vars` prefix.